data_processing_pipeline/6_visualize/src/build_svg.R

```diff
@@ -146,6 +149,32 @@ add_hover_rects <- function(svg_root, dat, state_nm, mx = 0, my = 0, scale_x = 1
   return(svg_root)
 }
 
+add_title <- function(svg_root) {
+  
+  svg_root %>% 
+    xml_add_child("text", id = "title-svg", transform = "translate(0 0)",
+                  style = "font-size: 14px; font-weight: 600; fill: rgb(139, 139, 139);",
```

Comment:
in the future, will probably want to put things like this in css instead of inline?

Reply:
I'm not sure what the ups or downs are? Just followed what @collnell had manually added. Colleen - do you know what the benefits/negatives are to doing it this way? I guess one benefit to using classes is that all of the styling can be found in one place.